mibios/glamr/views.py

```diff
@@ -906,19 +905,106 @@ def search(self, check_abundance=False):
                 if i.has_hit:
                     have_abundance = True
                     break
-            else:
-                have_abundance = False
-            model = content_type.model_class()
-            hits = [
-                (i.content_object, str(i.content_object), None)
-                for i in grp
-            ]
-            self.hits.append((
-                have_abundance,
-                model._meta.verbose_name_plural,
-                model._meta.model_name,
-                hits,
-            ))
+                else:
+                    have_abundance = False
+                model = content_type.model_class()
+                hits = [
+                    (i.content_object, str(i.content_object), None)
+                    for i in grp
+                ]
+                self.hits.append((
+                    have_abundance,
+                    model._meta.verbose_name_plural,
+                    model._meta.model_name,
+                    hits,
+                ))
```

Comment:
don't change the indentation, this is a for ... else block, the else does not belong to the if, the else block is skipped if the for loop is exited with a break.